tsconfig.json

```diff
@@ -11,8 +11,7 @@
     "outDir": "dist",
     "sourceMap": true,
     "strict": true,
-    "target": "es5",
-    "typeRoots": ["../node_modules/@types"]
```

Comment:
This folder (`../node_modules`) is outside of the repository, so I think it is a mistake. `node_modules/@types` is the default value, so we can just remove it.